docs/content/en/docs/reference/crd-reference/analysisdefinition.md

```diff
@@ -132,21 +139,21 @@ spec:
       target:
         failure:
           <operator>:
-            fixedValue: integer> |
+            fixedValue: <integer-quantity> |
```

Comment:
i don't think this has to be an integer, it can be anything that the quantity object allows